ansible/group_vars/all/inv_mapping.json

```diff
@@ -1,4 +1,4 @@
 {
    "lab": "lab_connection_graph.xml",
-   "": "// This is a comment.  specify all internal mappings below this line"
+   "inventory": "lab_connection_graph.xml"
```

Comment:
I believe this is a comment. can you specify it explicitly?

Reply:
Can you please explain more specifically what o you mean?
In json we can't add comments, so I removed empty line to avoid possible errors (if user will specify inventory name not like 'lab').

Reply:
The empty string as a key was added to specify the comments to the user on how this inv mapping is supposed to look like (since json doesn't support adding comments)
@yxieca do you have any suggestions on how this should look like. i believe you wanted this to avoid merge conflicts

Reply:
In any case empty string can't be defined here with such value, because code will fail if inventory name will have name other then 'lab', I have tested this and got issue, so thats I removed it.

Reply:
Maybe we can consider converting this inv_mapping.json to yaml format. Yaml file supports comments.

Reply:
If inventory name is something other than 'lab', it needs to be defined here along with its corresponding graph file. 
I suggest adding a comment line as follows 
"your_inventory_name" : "graph file for your inventory // sample inv graph mapping. Please specify all internal mappings below this line"
If you want to go the yaml route, I am okay with that too.

Reply:
In such case, lets move from json to yml file. I will add this change next week.

Reply:
I am ok with moving to yml. The 'comments' is to create enough gap so that github change won't always request manual merge when rebasing internal repos.

Reply:
@neethajohn @yxieca @wangxin Please take a look recent changes, thanks.